services/submission/src/main/java/app/coronawarn/server/services/submission/config/SubmissionServiceConfigValidator.java

```diff
@@ -42,14 +43,51 @@ public boolean supports(Class<?> type) {
   }
 
   /**
-   * Validate if the MaximumRequestSize of the {@link SubmissionServiceConfig} is in the defined range.
+   * Validates the following constraints.
+   * <ul>
+   *   <li>MaximumRequestSize is in the defined range.</li>
+   *   <li>List of SupportedCountries contains only valid ISO Codes</li>
+   *   <li>Mapping of trl (Transmission Risk Level) to dsos contains only values in allowed range</li>
+   *   <li>Mapping of dsos (Days Since Onset of Symptoms) to trl contains only values in allowed range</li>
+   * </ul>
    */
   @Override
   public void validate(Object o, Errors errors) {
     SubmissionServiceConfig properties = (SubmissionServiceConfig) o;
 
     validateMaxRequestSize(errors, properties);
     validateSupportedCountries(errors, properties);
+    validateDsosFromTrl(errors, properties);
```

Comment:
Based on previous PR reviews related to these fields i would rename the validation methods to:
- validateTransmissionRiskLevelDerivationMap
- validateDaysSinceSymptomsDerivationMap

Reply:
Made the change as suggested